app/jobs/education_form/create_daily_spool_files.rb

```diff
@@ -9,38 +11,65 @@ class CreateDailySpoolFiles
 
     WINDOWS_NOTEPAD_LINEBREAK = "\r\n"
 
-    def run(day = Date.yesterday)
-      # TODO: add and use the mapping of schools/regions -> spool file
+    attr_accessor :files
 
+    def run(day = Date.yesterday)
+      # Fetch all the records for the day
       records = EducationBenefitsClaim.unprocessed_for(day)
+      # Group the formatted records into different regions
+      regional_data = group_submissions(records)
+      # Create a remote file for each region, and write the records into them
+      create_files(day, regional_data)
+      # mark the records as processed
+      records.each { |r| r.update(processed_at: Time.zone.utc) }
```

Comment:
👍 I think I used `update_all` wrong at first somehow and convinced myself you couldn't use it on AR resultsets.